cmd/manager/main.go

```diff
@@ -151,6 +153,7 @@ func parseFlags() {
 	flag.Var(&managedNamespaceAnnots, "managed-namespace-annotation", "A regex indicating the annotations on namespaces that are managed by HNC. These annotations may only be set via the HierarchyConfiguration object. All regexes are implictly wrapped by \"^...$\". This argument can be specified multiple times. See the user guide for more information.")
 	flag.BoolVar(&webhooksOnly, "webhooks-only", false, "Disables the controllers so HNC can be run in HA webhook mode")
 	flag.BoolVar(&enableHRQ, "enable-hrq", false, "Enables hierarchical resource quotas")
+	flag.StringVar(&hncNamespace, "namespace", "hnc-system", "Namespace where hnc-manager and hnc resources deployed")
```

Comment:
BTW, hnc is going to make HA feature. I guess we will make leader selection by holding lock during specified namespace. So, does this arg use for it either?

Reply:
HNC HA isn't planning on using leader election for the time being. The "HA" part of HNC is the webhooks, which are stateless and so don't need to be leader elected; the controller will just restart. I'd rather put our effort into making it start up again 10x-100x faster than make the existing (slow) startup HA.